tests/environment-hook.bats

```diff
@@ -603,3 +603,21 @@ setup() {
 
   unstub vault
 }
+
+@test "Load custom json secret" {
+  export TESTDATA='{"MY_SECRET": "fooblah"}'
```

Comment:
i don't think this is testing the issue i initially reported. my issue was, the _value itself_ is a string of JSON. e.g.

`MY_SECRET: {"key": "some value", "anotherKey": "another value"}`
`MY_SECRET="{\"key\": \"some value\", \"anotherKey\": \"another value\"}"`